store/src/main/java/org/apache/rocketmq/store/MappedFileQueue.java

```diff
@@ -397,11 +397,21 @@ public int deleteExpiredFileByOffset(long offset, int unitSize) {
                         log.info("physic min offset " + offset + ", logics in current mappedFile max offset "
                             + maxOffsetInLogicQueue + ", delete it");
                     }
+                } else if (!mappedFile.isAvailable()) { // Handle hanged file.
```

Comment:
Why? I do not see the rationale of awaiting `RefCount`  to be 0 after commit log has been deleted after `intervalForcibly` period of time.